packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -658,6 +665,10 @@ - (void)interpretMapOptions:(NSDictionary *)data {
   if (myLocationButtonEnabled && myLocationButtonEnabled != (id)[NSNull null]) {
     [self setMyLocationButtonEnabled:[myLocationButtonEnabled boolValue]];
   }
+  NSString *style = data[@"style"];
+  if (style) {
```

Comment:
will this be NSNull?

Reply:
No, the corresponding JSON serialization code on the Dart side is:

```dart
if (config.style != null) 'style': config.style!,
```

Now that we are using in-package method channels everywhere, meaning that method channels in our plugins now map directly from Dart to a single implementation, I don't think there's much value in doing defensive coding against potential changes in Dart, because the only reason someone would change that Dart code would be to change this one other endpoint at the same time (unlike the old days, when someone might change the Dart code to do something for Android, not realizing that it would break iOS because they didn't even look).